.github/workflows/auto-publish.yml

```diff
@@ -42,14 +46,15 @@ jobs:
            tarBall: false
            zipBall: false
            preRelease: true
-           
+       - name: Sign mrpack
+         run: cosign sign-blob *.mrpack --bundle cosign.bundle --yes
```

Comment:
Modrinth does not currently accept the bundle on its own, but does accept a zip that contains it. Additionally, Skye made a feature request to support this file directly.